Doc/c-api/init.rst

```diff
@@ -1774,6 +1774,12 @@ Python-level trace functions in previous versions.
 
    The caller must hold the :term:`GIL`.
 
+.. c:function:: void PyEval_SetThreadProfile(PyThreadState* tstate, Py_tracefunc func, PyObject *obj)
```

Comment:
I dislike PyEval_ namespace. I suggest to rename it to: ``PyThreadState_SetProfile()``. Same for the Trace function.